indy_common/test/auth/multi_sig/test_auth_multi_sig_for_1_owner.py

```diff
@@ -16,8 +16,11 @@ def test_claim_def_adding_success_1_owner(write_request_validation, req,
 
 
 def test_claim_def_adding_success_2_owner(write_request_validation, req,
-                                          identity_owners, key):
+                                          identity_owners, endorsers, key):
     req.signatures = {idr: "signature" for idr in identity_owners[:2]}
+    # Endorser must be present in case of multi-sig
```

Comment:
Why? Before this PR this test showed, that signatures of two identity owners are enough for passing validation, so I suppose they had enough permissions. Now we have an endorser, and it is said in documentation that if original author of transaction have write access then endorser is not needed. Probably I'm missing something...

Reply:
This is because of a simplistic approach (see PoA) to require an endorser in case of any muti-signature if the author has unprivileged role. The motivation was because endorsement can be done in different ways by different roles and depends on Auth Constraints (probably composite ones). 
But you are right, I think we can easily enhance it to not require Endorser if there is a multi-signature from unprivileged roles only.

Reply:
Will do it in a next PR